admin/new.php

```diff
@@ -4,6 +4,11 @@ function gigpress_add() {
 
 	global $wpdb, $wp_locale;
 
+	/**
+	 * Filter to allow the venue city field to be set as not required.
+	 */
```

Comment:
```suggestion
	/**
	 * Filter to allow the venue city field to be set as not required.
	 *
 	 * @since TBD
 	 *
 	 * @param boolean If the city should be required.
 	 */
```